tony-history-server/startTHS.sh

```diff
@@ -179,9 +179,10 @@ save () {
 }
 
 # Set up TONY_CONF_DIR if not exist
+DEFAULT_TONY_CONF_DIR=/export/apps/tony/conf
```

Comment:
let's not put internal conf as default

Reply:
Ok, I'll change the default to `$APP_HOME/conf`.